pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -177,9 +177,10 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
         # PEP types key
         self.emit_title("PEP Types Key")
         for type_ in sorted(TYPE_VALUES):
-            self.emit_text(f"    {type_[0]} - {type_} PEP")
+            self.emit_text(f"* {type_[0]} - {type_} PEP")
```

Comment:
```suggestion
            self.emit_text(
                f"* **{type_[0]}** --- *{type_}*: ABBREVIATED_TYPES[type_]")
```

To add the short description of each type to the key, to make it actually useful

Also, needs to import `ABBREVIATED_TYPES` from `pep_headers` (which is possible now that it is a module-level dict constant :) )